packages/react/ds-core-form/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@canonical/react-ds-core-form",
```

Comment:
In the spec, this package is referred to as `@canonical/react-ds-addon-forms`, should this be renamed?

At the least, I suspect "core" should not be in the name unless we anticipate multiple React form packages.